managed/ui/src/components/tables/ListTables/ListTables.js

```diff
@@ -144,14 +144,15 @@ class ListTableGrid extends Component {
     const actions_disabled =
       isDisabled(currentCustomer.data.features, 'universes.backup') ||
       currentUniverse.universeDetails.backupInProgress;
+    const create_disabled = !(currentUniverse?.universeConfig?.takeBackups === 'true');
```

Comment:
Instead of putting `!` here, we can put it directly on the use case.

Reply:
Done